### PR TITLE
Framework for news section in HomeApp

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] #1253
[Problem] No news section
[Solution] Add framework for news section

### CODE REVIEW

**Comment by lslachciak** [comment]
image

**Comment by lslachciak** [comment]
In future we can fill this with JSON data from https://newsapi.org/s/google-news-api

**Review by TomaszLukawskiSam — APPROVED** [review]
Looks good to me.

File: examples/mobile/HomeApp/index.html [hunk 1]
```diff
@@ -42,9 +42,51 @@
...
+					<ul class="ui-listview">
+						<li>
+							Item 1
+						</li>
+						<li>
+							Item 2
+						</li>
+					</ul>
+				</li>
+				<li class="ui-expandable ui-li-divider">
```
Comment (HunseopJeong) [inline examples/mobile/HomeApp/index.html:75]: Can remove the ui-li-divider

**Review by HunseopJeong — APPROVED** [review]
LGTM. 1 minor comment.

File: examples/mobile/HomeApp/index.html (hunk 1 above)
Comment (lslachciak) [reply examples/mobile/HomeApp/index.html:75]: In guides there is divider between items so I would like to leave it
(see p. 92. 3-6 Expand Button of GUI Guide v2.2)

Comment (HunseopJeong) [reply examples/mobile/HomeApp/index.html:75]: Is the ui-li-divider drawing the underline of the list element?
image

**Pull request merged**